composer.json

```diff
@@ -0,0 +1,8 @@
+{
+    "require": {
+        "squizlabs/php_codesniffer": "*"
+    },
+    "scripts": {
+        "phpcs": "./vendor/bin/phpcs --config-set default_standard PSR2"
```

Comment:
Personally I dislike commands like `./vendor/bin/blahblahblah`.  I would put additional scripts in composer.json to wrap phpcs.  e.g. change the one you have to `phpcsinit` and add `phpcs` and `phpcbf` so all I need to do is type `composer phpcs /path/of/file`

Reply:
Good point! I'll try to make some changes